components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Engine/LegacyJobOrchestrator.cs

```diff
@@ -175,6 +178,15 @@ IJsonSerializer jsonSerializer
                         UpdatedJob = null,
                     };
                 }
+                if (info != null) {
+                    foreach (var assignedJob in _assignedJobs) {
+                    
```

Comment:
This line also contains trainilg whitespaces.

Reply:
Can I please ask you to install [Trailing Whitespace Visualizer](https://marketplace.visualstudio.com/items?itemName=MadsKristensen.TrailingWhitespaceVisualizer) Visual Studio extension which will automatically clear trailing whitespaces when the file is saved ?

Reply:
removed trailing whitespaces and installed Trailing Whitespace Visualizer, thanks